composer.json

```diff
@@ -30,24 +30,19 @@
 	"require-dev": {
 		"brain/faker": "dev-master",
 		"brain/monkey": "^2.6.1",
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
-		"infinum/eightshift-coding-standards": "^1.6",
+		"dealerdirect/phpcodesniffer-composer-installer": "^v1.0.0",
+		"infinum/eightshift-coding-standards": "2.0.0-beta",
+		"php-parallel-lint/php-parallel-lint": "^1.3",
 		"pestphp/pest": "^1.23.0",
 		"php-stubs/wordpress-stubs": "^6.3",
 		"phpunit/phpunit": "^9.6.8",
 		"szepeviktor/phpstan-wordpress": "^1.3",
 		"wp-cli/wp-cli": "^2.9"
 	},
-	"repositories": [
-		{
-			"type": "vcs",
-			"url": "https://github.com/infinum/eightshift-forms-utils.git"
-		}
-	],
 	"require": {
-		"php": "^7.4 || >=8.0",
+		"php": ">=8.2",
 		"erusev/parsedown": "^1.7.4",
-		"infinum/eightshift-forms-utils": "dev-feature/test-libs"
+		"infinum/eightshift-forms-utils": "dev-develop"
```

Comment:
The release should be stable before merging. Just adding a comment not to forget about this.

Reply:
this is only for develop. once we test everything before merging to main it will be update to the version stabile release